monai/networks/nets/vnet.py

```diff
@@ -67,16 +67,17 @@ def __init__(
     ):
         super().__init__()
 
-        if 16 % in_channels != 0:
+        if out_channels % in_channels != 0:
             raise ValueError(f"16 should be divisible by in_channels, got in_channels={in_channels}.")
```

Comment:
could you help fix the `16` in the error message?

also the project requires a sign-off line in the commit message https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#signing-your-work
please could you redo the commit message?

thanks!

Reply:
done fixed the error message!

Reply:
I signed off the latest commit. Is it required to sign off all commits?

Reply:
thanks, I'll merge this one, in the future please sign off all of them...